helpers/presentation_objects.py

```diff
@@ -98,6 +98,15 @@ def load(self, dst_folder, **kwargs):
 
         fig = plt.figure(figsize=(6, 4), dpi=150)
         ax = fig.add_subplot(1, 1, 1)
+
+        # show a (constant) reference value if it is defined
+        reference = kwargs.get("reference", None)
+        if reference:
+            ref_value = reference['value']
+            ref_label = reference['label']
```

Comment:
These will throw uncaught exceptions if "value" and "label" are not part of the provided `reference` dict (e.g., if somebody only provides a value since they do not know that this must be a dictionary). You could for instance use `get()` with reasonable default values or catch the `KeyError` and raise a `PresentationException`.

Reply:
Yes, looks good!